Makefile

```diff
@@ -20,11 +20,36 @@ install:
 	- mkdir -p /etc/sysusers.d/
 	cp distribution/osbuild-composer.conf /etc/sysusers.d/
 	systemd-sysusers osbuild-composer.conf
+	- mkdir -p /var/lib/osbuild-composer
+	- chown _osbuild-composer:_osbuild-composer /var/lib/osbuild-composer
```

Comment:
Should this also be added to the specfile?

Reply:
This was a mistake - `/var/lib/osbuild-composer` is created by systemd when starting `osbuild-composer.service`.